src/go/rpk/pkg/cli/cmd/wasm/generate.go

```diff
@@ -63,8 +63,8 @@ func NewGenerateCommand(fs afero.Fs) *cobra.Command {
 	}
 
 	command.Flags().StringVar(
-		&dir,
-		"dir",
+		&output,
+		"output",
```

Comment:
I've seen flags that represent similar things called `--out` or `--out-dir`. `--output` is kind of confusing, so I would suggest renaming it to any of the previous 2.

Reply:
Got it, I used `--out-dir`